tests/unit_tests/commands/android/testIndex.js

```diff
@@ -440,6 +440,21 @@ describe('test getConfigFromOptions', function() {
 
     configs = androidSetup.getConfigFromOptions({mode: true, browsers: []});
     assert.deepStrictEqual(configs, {});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true});
+    assert.deepStrictEqual(configs, {browsers: 'none'});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true, browsers: false});
+    assert.deepStrictEqual(configs, {browsers: 'none'});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true, browsers: 'chrome'});
+    assert.deepStrictEqual(configs, {browsers: 'chrome'});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true, browsers: 'chrome,firefox'});
+    assert.deepStrictEqual(configs, {browsers: 'both'});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true, browsers: 'brave'});
+    assert.deepStrictEqual(configs, {});
```

Comment:
In this case, the `browser` should be set to `none` because we don't want to ask the browser question in this case also.